src/ClientOperations/AbstractOperation.php

```diff
@@ -93,6 +104,7 @@ public function createNetworkPackage(string $correlationId): TcpPackage
     public function inspectPackage(TcpPackage $package): InspectionResult
     {
         if ($package->command()->equals($this->responseCommand)) {
+            /** @psalm-suppress InvalidStringClass */
             $responseMessage = new $this->responseClassName();
```

Comment:
The `responseClassName()` method should have a `@return class-string<...>` https://psalm.dev/docs/annotating_code/type_syntax/scalar_types/

Reply:
done